Atomic/util.py

```diff
@@ -209,7 +214,9 @@ def skopeo_inspect(image, args=[]):
     except OSError:
         raise ValueError("skopeo must be installed to perform remote inspections")
     if results.return_code is not 0:
-        raise ValueError(results.stderr)
+        # Need to check if we are dealing with a v1 registry
+        is_v1_registry(image)
+        raise ValueError("Unable to interact with this registry: {}".format(results.stderr))
     else:
```

Comment:
Why do you handle this error differently?  
Could  you change function call to check_v1_registry, or try_v1_registry.  
is_ sounds like a boolean check.